src/irule.cpp

```diff
@@ -322,11 +325,25 @@ main( int argc, char **argv ) {
                 if ( rulegen ) {
                     if ( startsWith( buf, "INPUT" ) || startsWith( buf, "input" ) ) {
                         gotRule = 1;
-                        trimSpaces( trimPrefix( buf ) );
+                        constexpr auto skip_len{5};
+                        const auto copy_length{std::strlen(buf) + 1 - skip_len};
+
+                        // Blank out "INPUT"
+                        std::memmove(buf, buf + skip_len, copy_length);
+
+                        auto res{boost::algorithm::trim_left_copy(std::string{buf})};
+                        std::strncpy(buf, res.c_str(), res.size() + 1);
```

Comment:
Consider adding `constexpr auto null_term{1};` to make the intent of the `+ 1` more obvious.

Possibly a better name for that too?

Same for other block.

Reply:
Done.